### DISCUSSION TITLE
Multiple requests handler

### DISCUSSION THREAD

**Discussion by iamxorum** [Zero6992/chatGPT-discord-bot#355] [Ideas]
Hey! I had an Idea, what about an update to make the bot to handle multiple requests, I mean, If are made 2 or requests in a short time, what about to make a queue and start to response them in FIFO method?

**Comment by Zero6992** [comment]
Thank you for the suggestion. It has been supported in the latest version.

**Reply by iamxorum** [reply]
Thank you, you're doing a sensational job! I would like to know How you learned Python so well o.o
